hazelcast/src/main/java/com/hazelcast/crdt/AbstractCRDTReplicationOperation.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.crdt;
+
+import com.hazelcast.internal.partition.MigrationCycleOperation;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.Operation;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static com.hazelcast.util.MapUtil.createHashMap;
+
+/**
+ * Base class for CRDT replication operations. It support replicating a
```

Comment:
minor typo: `it supports replicating...`

Reply:
👍